Makefile

```diff
@@ -1,13 +1,16 @@
 ASSETS := $(shell yq e '.assets.[].src' manifest.yaml)
 ASSET_PATHS := $(addprefix assets/,$(ASSETS))
 DOC_ASSETS := $(shell find ./docs/assets)
-VERSION_TAG := $(shell ./get_tag.sh)
-VERSION := $(shell ./get_tag.sh | cut -c 2-)
+#VERSION_TAG := $(shell ./get_tag.sh)
```

Comment:
This had to be done because of an issue where the btcpay team messed up the git tag/commit process.